frontend/src/__tests__/integration/pages/modelServing/ModelServingGlobal.spec.ts

```diff
@@ -30,7 +30,8 @@ test('Edit model', async ({ page }) => {
   await await page.getByLabel('Model Name *').fill('');
   await await page.getByLabel('Path').fill('');
   await expect(await page.getByRole('button', { name: 'Deploy', exact: true })).toBeDisabled();
-
+  await await page.getByLabel('Path').fill('/');
```

Comment:
Not sure if this is right. Should I use a `Promise.race()` for both `/` and ` ` inputs instead? Any other tests to add?